src/MoBi.Presentation/Tasks/ProjectTask.cs

```diff
@@ -275,33 +276,47 @@ private bool loadProjectIntoContext(string fileName)
             return false;
 
          _mruProvider.Add(fileName);
-
+         
          notifyProjectLoaded();
+
          return true;
       }
 
       private void generateDefaultsInCurrentProject()
       {
-         addDefault<IMoleculeBuildingBlock>(AppConstants.DefaultNames.MoleculeBuildingBlock);
-         addDefault(AppConstants.DefaultNames.ReactionBuildingBlock, () => _reactionBuildingBlockFactory.Create());
-         addDefault(AppConstants.DefaultNames.SpatialStructure, () => _spatialStructureFactory.CreateDefault(AppConstants.DefaultNames.SpatialStructure));
-         addDefault<IPassiveTransportBuildingBlock>(AppConstants.DefaultNames.PassiveTransportBuildingBlock);
-         addDefault<IEventGroupBuildingBlock>(AppConstants.DefaultNames.EventBuildingBlock);
-         addDefault<IObserverBuildingBlock>(AppConstants.DefaultNames.ObserverBuildingBlock);
+         var module = _context.Create<Module>();
```

Comment:
Why is the default with everythung in it?

Reply:
I think the workflow here is the same as 'New Project' with Building Blocks. You need at least one of each type to make anything right?

That's the flow I tried to preserve.